src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
-public class Duke {
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class
+Duke {
```

Comment:
Please bring your Duke to the same line 
```suggestion
public class Duke {
```